nss-day-cohort-22/Class-Website
# Two small fixes

## Link to Ticket

TITLE [Link]()

## Description of Proposed Changes

- added http:// to Jesse's website link so it would properly load when clicked
- removed the `:` from behind Greg's name on the list of thank yous.


## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-WebsiteLinkFixTypoFix
```

Load site via Http-server
Navigate to Jesse's profile and click the link to her website. Make sure her page loads successfully in a new tab or window

## Impacted Areas in Application

List general components of the application that this PR will affect:

* cohort.json
* Index.html

## Mentions @username

Tag users that need to review this code
